app-fsharp-xaml/Program.fs

```diff
@@ -0,0 +1,30 @@
+﻿namespace AvaloniaAppTemplate
+
+open Avalonia
+open Avalonia.Logging.Serilog
+open Avalonia.Controls
+open Avalonia.Markup.Xaml
+
+type MainWindow() as this =
+    inherit Window()
+
+    do
+        AvaloniaXamlLoader.Load(this)
+        
+type App() =
+    inherit Application()
+    
+    override this.Initialize () =
+        let x = this.GetType()
+        AvaloniaXamlLoader.Load(this)
+ 
+module Program =
+
+    [<EntryPoint>]
+    let main argv =
```

Comment:
1) this doesn't work with previewer, a separate BuildAvaloniaApp static method is needed
2) The preferred approach is a separate AppMain to ensure proper initialization order, see example here: https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/app/Program.cs

Reply:
Is resolved - previewer (VS 2019) works now